send/send.c

```diff
@@ -95,16 +95,24 @@ static void append_signature(FILE *fp, struct ConfigSubset *sub)
   pid_t pid;
 
   const char *c_signature = cs_subset_path(sub, "signature");
-  if (c_signature && (fp_tmp = mutt_open_read(c_signature, &pid)))
+  if (!c_signature)
+    return;
+
+  fp_tmp = mutt_open_read(c_signature, &pid);
```

Comment:
Can you please change this to a definition, too? `FILE *fp_tmp = mutt_open_read(c_signature, &pid);`